imagej/imagej-ops2/pom.xml

```diff
@@ -210,7 +210,11 @@
 		<license.projectName>ImageJ software for multidimensional image processing and analysis.</license.projectName>
 
 		<!-- NB While we depend on old scijava-common -->
-		<imagej-ops2.allowedDuplicateClasses>${scijava.allowedDuplicateClasses}, org.scijava.*,com.github.therapi.runtimejavadoc.repack.com.eclipsesource.json.*</imagej-ops2.allowedDuplicateClasses>
+		<imagej-ops2.allowedDuplicateClasses>
+			${scijava.allowedDuplicateClasses},
```

Comment:
Doesn't the whitespace hose this up? I tried making a similar change in pom-scijava, and got output like:
```
[INFO] Adding ignore: com.esotericsoftware.kryo.*
[INFO] Adding ignore:
			com.esotericsoftware.minlog.*
[INFO] Adding ignore:
			com.esotericsoftware.reflectasm.*
[INFO] Adding ignore:
			com.google.inject.*
[INFO] Adding ignore:
			jnr.ffi.*
[INFO] Adding ignore:
			org.apache.hadoop.yarn.*.package-info
[INFO] Adding ignore:
			org.apache.spark.unused.UnusedStubClass
[INFO] Adding ignore:
			org.hibernate.stat.ConcurrentStatisticsImpl
[INFO] Adding ignore:
			org.jetbrains.kotlin.daemon.common.*
[INFO] Adding ignore:
			org.junit.runner.Runner
```
When before it looked like:
```
[INFO] Adding ignore: com.esotericsoftware.kryo.*
[INFO] Adding ignore: com.esotericsoftware.minlog.*
[INFO] Adding ignore: com.esotericsoftware.reflectasm.*
[INFO] Adding ignore: com.google.inject.*
[INFO] Adding ignore: jnr.ffi.*
[INFO] Adding ignore: org.apache.hadoop.yarn.*.package-info
[INFO] Adding ignore: org.apache.spark.unused.UnusedStubClass
[INFO] Adding ignore: org.hibernate.stat.ConcurrentStatisticsImpl
[INFO] Adding ignore: org.jetbrains.kotlin.daemon.common.*
[INFO] Adding ignore: org.junit.runner.Runner
```
So we might need to squash this back together?

Reply:
Fixed in 2070792c